cmd/kyma/alpha/create/module/module.go

```diff
@@ -195,6 +195,15 @@ func (cmd *command) Run(args []string) error {
 
 func (cmd *command) validateDefaultCR(modDef *module.Definition, l *zap.SugaredLogger) error {
 	cmd.NewStep("Validating Default CR")
+
+	if modDef.DefaultCRPath != "" {
```

Comment:
This prefill DefaultCR is also done in `inspectProject` project step https://github.com/kyma-project/cli/blob/main/pkg/module/resources.go#L213-L226 , but have the precondition `if def.RegistryURL != "" `, I don't have too much detail context for this decision. IMO, if this `DefaultCR` in anyway needs to be filled when `DefaultCRPath` exits, then it's better remove that precondition in `module.Inspect`, this step is happen before `validateDefaultCR`